pixi/src/ui/recommendations/RecommendationsView.js

```diff
@@ -83,11 +87,22 @@ export default class RecommendationsView {
         bodyHtml = bodyHtml.replace(/\$\{URL\}/g, encodeURIComponent(pageURL));
         bodyHtml = addTargetBlankToLinks(bodyHtml);
 
-        body.innerHTML = bodyHtml;
+        bodyText.innerHTML = bodyHtml;
+
+        // Set 'next advice' button
+        const nextRecommendation = recommendations[i + 1];
+        if (nextRecommendation) {
+          nextButton.href = `#recommendation-${nextRecommendation.id}`;
+          nextButton.addEventListener('click', () => {
+            this.onClickNext(recommendation);
+          });
+        } else {
+          nextButton.remove();
+        }
       }
 
       recommendation.style = null;
-      recommendation.id = `recommendation-${value.id}`;
+      title.id = `recommendation-${value.id}`;
```

Comment:
Is it OK if we restore it to set/link to the `recommendation.id` instead of `title.id`? I understand we need more padding to account for the top banners, however as it is the screen reader will get trapped on the title element after clicking "Next advise"